examples/example-gcp-csm-observability/README.md

```diff
@@ -0,0 +1,43 @@
+gRPC GCP CSM Observability Example
+================
+
+The GCP CSM Observability example consists of a Hello World client and a Hello World server and shows how to configure CSM Observability
+for gRPC client and gRPC server.
+
+## Configuration
+
+`CsmObservabilityClient` takes the following command-line arguments -
+* user - Name to be greeted.
+* target - Server address. Default value is `xds:///helloworld:50051`.
+  * When client tries to connect to target, gRPC would use xDS to resolve this target and connect to the server backend.
+* prometheusPort - Port used for exposing prometheus metrics. Default value is `9464`.
+
+
+`CsmObservabilityServer` takes the following command-line arguments -
+* port - Port used for running Hello World server. Default value is `50051`.
+* prometheusPort - Port used for exposing prometheus metrics. Default value is `9465`.
```

Comment:
should probably default to 9464

Reply:
`PrometheusHttpServer` creates a Http server, if both client and server use the same port on same machine, whichever is run later will get an exception. That is the reason, I opted to use two different ports.

Reply:
if the client and the server were on the same machine/pod/vm instance, there wouldn't be much of a reason to use CSM

Reply:
Updated.